code/modules/mining/mine_turfs.dm

```diff
@@ -255,15 +255,14 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 
 		playsound(user, P.drill_sound, 20, 1)
 
-		//handle any archaeological finds we might uncover
 		var/fail_message = ""
+		//handle any archaeological finds we might uncover
 		if(finds && finds.len)
 			var/datum/find/F = finds[1]
-			if(excavation_level + P.excavation_amount > F.excavation_required)
 
-				fail_message = ", <b>[pick("there is a crunching noise","[W] collides with some different rock","part of the rock face crumbles away","something breaks under [W]")]</b>"
-
-		to_chat(user, "<span class='rose'>You start [P.drill_verb][fail_message].</span>")
+			if(excavation_level + P.excavation_amount > F.excavation_required)
```

Comment:
Why is GitHub's diff so shit

You just moved this down a line, right? That's what it looks like. I'm just confused because the diff is being weird.